src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -2991,3 +2986,78 @@ instruct vcvtDtoB(vReg dst, vReg src, vReg tmp)
   %}
   ins_pipe(pipe_slow);
 %}
+
+// ------------------------------ Vector extract ---------------------------------
+
+instruct extractB(iRegINoSp dst, vReg src, immI idx, iRegINoSp tmp,  pRegGov pTmp, rFlagsReg cr)
+%{
+  predicate(UseSVE > 0 && n->in(1)->bottom_type()->is_vect()->length_in_bytes() >= 16);
+  match(Set dst (ExtractB src idx));
+  effect(TEMP tmp, TEMP pTmp, KILL cr);
+  ins_cost(2 * SVE_COST);
+  format %{ "movzw $tmp, $idx\n\t"
+            "sve_whilele $pTmp, B, zr, $tmp\n\t"
+            "sve_lastb $dst, B, $pTmp, $src\n\t"
+            "sbfmw $dst, $dst, 0U, 7U\t# extract from vector(B)" %}
+  ins_encode %{
+    __ movzw(as_Register($tmp$$reg), (int)($idx$$constant));
+    __ sve_whilele(as_PRegister($pTmp$$reg), __ B, zr, as_Register($tmp$$reg));
```

Comment:
I think we can use rscratch1 here to save one register. See https://github.com/openjdk/panama-vector/pull/49

Reply:
OK. Thank you for your review.I will fix it in my next patch. For other `$tmp` in `adfile`, I will create a new issue (JDK-8263578) and try to replace them.